### PR TITLE
Update the translated post when translated strings are recieved

### PR DESCRIPTION
**Pull request by brucepearson**
Update the translated post when translated strings are recieved - https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-5643

### CODE REVIEW

**Comment by brucepearson** [comment]
@OnTheGoSystems/wpml Please review

**Review by andreasciamanna — APPROVED** [review]

File: src/class-wpml-gutenberg-integration.php [hunk 1]
```diff
@@ -46,6 +46,40 @@ function register_strings( WP_Post $post, $package_data ) {
...
+		$translated_post_id,
+		$original_post,
+		$string_translations,
+		$lang
+	) {
+
+		if ( self::PACKAGE_ID === $package_kind ) {
+			$blocks = gutenberg_parse_blocks( $original_post->post_content );
+
+			foreach ( $blocks as &$block ) {
```
Comment (adrianosferreira) [inline src/class-wpml-gutenberg-integration.php:61]: @BruceOnTheGo why as reference?

**Review by adrianosferreira — APPROVED** [review]
Just a question, looks good +1

**Comment by brucepearson** [comment]
Merged to wpmltm-2112 branch

**Pull request closed without merging**